packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -225,6 +226,18 @@ mixin SchedulerBinding on BindingBase, ServicesBinding {
   AppLifecycleState get lifecycleState => _lifecycleState;
   AppLifecycleState _lifecycleState;
 
+  /// Initializes the [lifecycleState] with the data from the window.
+  ///
+  /// An optional [testWindow] may be provided to supply test values.
```

Comment:
Would you mind including enough information here to direct the reader to where/how to use a test Window?  Technically one is not "supplied", and I think the use of a test Window should only be done from within a test with a test binding, right?

Reply:
This comment is actually left over from the previous version and does not apply anymore. Will modify to refect the correct way to pass test values.